### DISCUSSION TITLE
更新日志 - 22.03.7 - 20241206

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1960] [Announcements]
# 系统
更新 r8125 驱动到最新版本，并修复某些奇怪的Device ID（10ec:8162）降级到千兆的问题

# 应用
* QuickStart: 增加LAN口保活机制：
   1. 当WAN口IP跟LAN口冲突时自动修改LAN口IP（192.168.100.1或192.168.101.1）；
   2. 当联网失败，并且非旁路由模式，且LAN口有其他DHCP服务，则从这个DHCP服务获取IP和网关，检测这个网关能访问 223.5.5.5 则使用此IP和网关上网（会启动一个planb接口）；
   3. 在不冲突的前提下额外启动一个192.168.100.1，使用户能通过配置静态IP访问192.168.100.1后台。
   4. 对于高级用户而言，如果这些功能影响使用，可以在iStore中安装或更新“系统便利工具”，执行其中的“禁用 LAN 口保活”即可关闭。

# 设备
* Easepi R1：新增支持我们的新硬件
* T68m：更新千兆网口LED灯配置

**Comment by zzzbaba** [comment]
憋了又1个月以为是个大更

**Reply by jannson** [reply]
超大的更新，其实。特别是 LAN 口保活，以及其他一些配套软件的更新。

**Comment by yoyo1001** [comment]
希望增加wan6口的定时重启功能，或者可以自动监测，只要是断了就重启。场景是：一些光猫拨号的ipv6网络，在看似正常的情况下，实际已经失去了IPv6的联网，需要手动重启wan6，才能恢复ipv6联网。

**Reply by jianyuenet** [reply]
Yes you are right. Now we can only reboot iStoreOS to solve the problem.

**Reply by Phylirui** [reply]
我在计划任务里面加了这条，每周二四六早上4点重启WAN6
0 4 * * 2,4,6 ifup wan6

**Reply by jjm2473** [reply]
我家的ipv6出问题的话，要重启光猫才能恢复正常

**Comment by FrankkX** [comment]
更新系统便利工具后，“禁用 LAN 口保活”无法执行。

**Reply by cbcbccbb** [reply]
我也是，更新完这个需要重启吗？

**Reply by chinesepy** [reply]
好像是systools.sh 脚本没更新，不知道手动执行下/usr/share/systools/disable-planb.run 行不行
另，istoreos.com域名从国外又无法访问了...

**Reply by jannson** [reply]
手动执行也可以的。本质就是执行这个脚本。

**Reply by jjm2473** [reply]
再更新下就行了，之前没在前置脚本里引用

**Comment by yoyo1001** [comment]
还有能否测试解决一下，docker非hosts模式下导致ipv6不能联网的问题。主要表现为，docker不是hosts模式的时候，ipv6连接看似正常，实际已经无法访问ipv6网络。

**Comment by itgamer** [comment]
什么时候适配23，一直停留在22

**Reply by bcseputetto** [reply]
不会适配23.05，下个大版本应该是24.10

**Reply by itgamer** [reply]
> 不会适配23.05，下个大版本应该是24.10

Realy?

**Reply by jjm2473** [reply]
x86有23.05的测试版本，但是不打算继续适配了，因为内核版本接近，该不支持的硬件还是不支持，适配了也是浪费时间。下个版本会用24.10

**Comment by gtxy27** [comment]
24.10有预计会什么时候上，有些插件像homeproxy 装不了

**Comment by lichao0223** [comment]
24.10 已经rc2了，会跟进出个测试版么

**Comment by cyberwall404** [comment]
什么时候把luci升级下，科学节点列表展示太慢了

**Reply by jjm2473** [reply]
其他固件，使用同一个翻墙插件的情况下，科学节点列表展示能更快吗？
据我所知，例如passwall还是在使用lua渲染页面（为了兼容19.07及更早的openwrt），不管你用什么固件都会加载缓慢。

**Reply by cyberwall404** [reply]
之前用esir的版本，列表页面加载很快，不过那个passwall版本比较低，节点自动切换还不是本地socket分组这种方式，不清楚是不是和版本有关系。

以前一直以为是硬件配置太低，软路由从rs5，换成x86，发现都很慢。
日常使用影响倒也不是特别大，主要在于，机场用户，节点特别多，不小心点到要么等半天加载，要么关掉重新打开。

**Reply by bcseputetto** [reply]
@cyberwall404 慢的原因不是OpenWrt的luci没有升级，而是插件还在用 lua，没有切到 js

**Comment by smhjw** [comment]
新版本内存占用异常的高

**Reply by jjm2473** [reply]
未发现异常。什么机型，更新前后内存占用差别多大

**Comment by wcbspp** [comment]
烦问作者~  istoreo 开启ipv6的情况下，MTU 和 MSS需要特别的设置嘛
我目前的情况是PC访问网站都正常，只有手机端刷抖音会加载非常缓慢，应该是和V6的DNS解析有关系？ 调整了一下目前还是存在加载缓慢的问题。

**Comment by changandeyue** [comment]
作者您好，设备R5S，istoreos首页-网络接口状态-变成空白的了，网络-网口配置也变成丢失状态。版本是22.03.7 20240802。我在升级至最新版(20241206)后问题依旧存在。在查阅以往commemt好像不仅我一个人遇到这个问题，具体的情况就像这样https://github.com/istoreos/istoreos/discussions/1869#discussioncomment-11434956

**Reply by jjm2473** [reply]
开个issue讨论。安装istore里面的系统便利工具，然后执行其中的修复系统软件，把修复过程日志发出来看看。

**Reply by bcseputetto** [reply]
@changandeyue 如果可以，请你提供一下终端/ssh 执行
```console
uci show |grep firewall.@zone
```
的执行结果

**Reply by jjm2473** [reply]
@bcseputetto 跟防火墙应该没啥关系。根据提供的链接，应该是quickstart的一个api（`/cgi-bin/luci/istore/network/interface/config/`）执行命令失败，执行的命令很可能是`ip -pretty -detail -json link show`

**Reply by jjm2473** [reply]
这边有记录排查过程和解决方案 https://github.com/istoreos/istoreos/issues/1986

**Comment by Aupinda** [comment]
这个版本树莓派无法启动
树莓派使用USB接了SSD启动，参考这里 https://github.com/QuentinHsu/Document-Page/issues/12  修改了cmdline.txt
前面版本正常使用，目前在用 iStoreOS 22.03.7 2024080210

**Reply by jjm2473** [reply]
这段时间没改rpi4的内核配置，如果有问题，提个issue，把完整的启动日志传上来。

**Comment by 775265504** [comment]
<img width="5376" height="1928" alt="ScreenShot_2025-12-22_232606_365" src="https://github.com/user-attachments/assets/ffa9b7fd-12fa-429b-8cc1-cb0cd637c4ce" />
记得更换过主题，然后恢复到默认argon主题后，访问软路由管理页面，便不能自动跳转到quickstart页面了，访问软路由地址停留在http://192.168.100.1/cgi-bin/luci/，必须要手动点一下首页才能正常显示首页的软路由状态概览网址http://192.168.100.1/cgi-bin/luci/admin/quickstart/。我尝试用过恢复早期全量备份都不能恢复，以及尝试使用过系统便利工具里面的修复系统软件也没有用。请问这种跳转逻辑丢失或者错误的原因是什么？请问要如何恢复？
